lib/src/components/NxList/types.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import {
+  HTMLAttributes,
+  LiHTMLAttributes,
+  ValidationMap,
+  ReactNode,
+  ButtonHTMLAttributes,
+  AnchorHTMLAttributes
+} from 'react';
+import PropTypes from 'prop-types';
+
+export interface NxListProps extends HTMLAttributes<HTMLUListElement> {
+  bulleted?: boolean | null;
+  children?: ReactNode;
```

Comment:
Probably redundant with the parent type. Same with the other interfaces below

Reply:
> Probably redundant with the parent type. Same with the other interfaces below

`children` has been removed.
1776aaebcc703414808783c567c01b0a222ce75f